scripts/configure-cluster/common/setup-postgres-operator.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -x
```

Comment:
Please move this set to the script including thsi one, as a developer will want to include this file in its bash to run the function manually, and that will make its bash very verbose ;)

Reply:
ack done